tests/equation_bcs/test_bcs.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from firedrake import *
+
+
+@pytest.mark.parallel
```

Comment:
Does this test need to be parallel? Prefer not, since they are much slower.

Reply:
No, it does not.  Fixed.

Reply:
I don't see these changes.

Reply:
Sorry, just pushed.